README.md

```diff
@@ -90,6 +90,7 @@ The following 61 devices are supported.
 | [MAX7219 & MAX7221 display driver](https://datasheets.maximintegrated.com/en/ds/MAX7219-MAX7221.pdf) | SPI |
 | [MCP23017 port expander](https://ww1.microchip.com/downloads/en/DeviceDoc/20001952C.pdf) | I2C |
 | [MCP3008 analog to digital converter (ADC)](http://ww1.microchip.com/downloads/en/DeviceDoc/21295d.pdf) | SPI |
+| [MCP2515 Stand-Alone CAN Controller with SPI Interface](https://ww1.microchip.com/downloads/en/DeviceDoc/MCP2515-Family-Data-Sheet-DS20001801K.pdf) | SPI |
```

Comment:
Please update the numbers as the number of devices has increased.

```diff
-The following 61 devices are supported.
+The following 62 devices are supported.
```

Reply:
Actually now it will be 63, if this is able to be merged next.

Reply:
And now it would be number 64 `2^6` which is a very fortuitous number indeed.

Reply:
updated.   I counted it and listed it as 64.